README.md

```diff
@@ -57,6 +57,76 @@ them and then do:
 
     % git subrepo push ext/os-autoinst-common
 
+### Possible Workflows
+
+One can make changes to the subrepo directly from the host repo, but there are
+different strategies.
+
+#### Make changes in host repo first and push after approval
+
+* Create a branch in the host repo
+* Make changes in the subrepo directory
+* Create PR
+* After approval, merge and push changes to subrepo
+
+
+Prepare:
+
+    % git checkout -b feature1
```

Comment:
I would prefer no `%` in code blocks for easier copying into terminals :)

Reply:
The old-school `%` prompt is so far being used everywhere. I'd suggest to change this separately.

Reply:
Regardless which shell you use and what prompt character it uses, if it is `$` for a user-bash, `#` for a root-bash, `%` in zsh or `>` in tcsh, please use shell code sections without prefixes for easier copying. Compare to https://github.com/os-autoinst/scripts/blob/master/README.md#openqa-label-known-issues